dubbo-filter/dubbo-filter-validation/src/main/java/org/apache/dubbo/validation/support/jvalidation/JValidatorNew.java

```diff
@@ -293,7 +292,7 @@ public void validate(String methodName, Class<?>[] parameterTypes, Object[] argu
             }
         } catch (ValidationException e) {
             // only use exception's message to avoid potential serialization issue
-            throw new RpcException(RpcException.VALIDATION_EXCEPTION, e.getMessage(), e);
+            throw new ValidationException(e.getMessage());
```

Comment:
I'm a little concerned about affecting users who rely on `RpcException.VALIDATION_EXCEPTION`

Reply:
RpcException.VALIDATION_EXCEPTION is bought by https://github.com/apache/dubbo/pull/9552/files#diff-d328eb1d6b3db5068164007286dffcd60b8419b3bfa0d7ebc566635f39f5b60fR42, which has not been published